var/spack/repos/builtin/packages/visit/package.py

```diff
@@ -0,0 +1,32 @@
+from spack import *
+
+
+class Visit(Package):
+    """VisIt is an Open Source, interactive, scalable, visualization, animation and analysis tool."""
+    homepage = "https://wci.llnl.gov/simulation/computer-codes/visit/"
+    url = "http://portal.nersc.gov/project/visit/releases/2.10.1/visit2.10.1.tar.gz"
+
+    version('2.10.1', '3cbca162fdb0249f17c4456605c4211e')
+
+    depends_on("vtk@6.1.0~opengl2")
+    depends_on("qt@4.8.6")
+    depends_on("python")
+    depends_on("hdf5")  # silo seems to need it
+    depends_on("silo+shared")
+
+    def install(self, spec, prefix):
+        with working_dir('spack-build', create=True):
+
+            feature_args = std_cmake_args[:]
+            feature_args = ["-DVTK_MAJOR_VERSION=6",
+                            "-DVTK_MINOR_VERSION=1",
+                            "-DCMAKE_INSTALL_PREFIX:PATH=%s" % spec.prefix,
```

Comment:
this is most likely there in `std_cmake_args`, you can check `spack_build.out` to be sure.